src/components/LibraryItem.tsx

```diff
@@ -258,6 +263,14 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
         }
     }
 
+    // Unexpand all other slibling items when one item is being expanded
```

Comment:
Wait, I think this does something entirely different. Let's rephrase it this way:

```js
// Collapse all child items when one of the child items is expanded
onSingleChildItemExpanded() {
  for (...) { }
  this.setState({ expanded: true }); // Make the current item (parent) expanded.
}
```